SimpleITK/utilities/fft.py

```diff
@@ -20,7 +20,12 @@
 
 
 def fft_based_translation_initialization(
-    fixed: sitk.Image, moving: sitk.Image
+    fixed: sitk.Image,
+    moving: sitk.Image,
+    *,
+    required_fraction_of_overlapping_pixels: float = 0.0,
+    initial_transform: sitk.Transform = None,
+    maked_pixel_value: float = None,
```

Comment:
missing s `masked_pixel...` not `maked_pixel...` auto complete ensured the spelling mistake was consistent in the documentation and code, so requested change is not limited to this line.

Reply:
O that change locally.